staging/src/k8s.io/apimachinery/pkg/apis/meta/internalversion/types.go

```diff
@@ -45,4 +46,16 @@ type ListOptions struct {
 	ResourceVersion string
 	// Timeout for the list/watch call.
 	TimeoutSeconds *int64
+
+	From  string
+	Limit int64
```

Comment:
+doc for `From` and `Limit` here for consistency with the other `ListOptions` fields?  I see they're also documented nicely in types.go.